clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -105,6 +105,10 @@ Improvements to clang-tidy
   now run checks in parallel by default using all available hardware threads.
   Both scripts display the number of threads being used in their output.
 
+- :program:`clang-tidy` no longer attemps to match AST nodes from system headers
+  by default, greatly improving performance. This behavior is disabled if the
```

Comment:
`AST nodes` might be too much of a compiler term. `no longer attempts to analyze code from system headers` might be better?